build/mavenBuildDeps/buildArtifactsAGVList/BuildArtifactsAGVListTest.groovy

```diff
@@ -1,19 +1,71 @@
 import spock.lang.Specification
-
-import static org.jfrog.artifactory.client.ArtifactoryClient.create
+import groovy.json.JsonSlurper
+import org.jfrog.artifactory.client.ArtifactoryClient.*
 
 class BuildArtifactsAGVListTest extends Specification {
-    def 'test name'() {
+    static final baseurl = 'http://localhost:8088/artifactory'
+    static final auth   = "Basic ${'admin:password'.bytes.encodeBase64()}"
+    static final build_info_file   = 'build-info.json'
+
+    def 'test build artifacts AGVList '() {
         setup:
-        def baseurl = 'http://localhost:8088/artifactory'
-        def artifactory = create(baseurl, 'admin', 'password')
+
+        def conn = new URL("${baseurl}/api/build").openConnection()
+        conn.requestMethod = 'PUT'
+        conn.doOutput = true
+        conn.setRequestProperty('Authorization', auth)
+        conn.setRequestProperty('Content-Type', 'application/json')
+        def buildInfoFile = new File("./src/test/groovy/${build_info_file}")
```

Comment:
@davidshinjfrog from the plugin's root folder, both test case and `build-info.json` files should be inside a folder called `test`. Then, the test case should use the path below to get to the `build-info.json` file:

```groovy
def buildInfoFile = new File("./src/test/groovy/BuildArtifactsAGVListTest/${build_info_file}")
```

This must be done so the framework can automatically execute this test on our CI environment. During load phase, the framework copy everything from the test folder to that path before running the test case.